src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -49,18 +49,51 @@
       <DataSource Persistence="ProjectFile" PersistedName="Win32Resource" />
     </StringProperty.DataSource>
   </StringProperty>
-  <StringListProperty Name="AvailablePlatforms" Separator="," Visible="False" />
-  <StringProperty Name="FileName" DisplayName="Project File" ReadOnly="True">
+    <StringProperty Name="DefineConstants" DisplayName="Define Constants"/>
+    <EnumProperty Name="PlatformTarget" DisplayName="Platform Target"/>
+    <StringProperty Name="Prefer32Bit" DisplayName="Prefer 32Bit"/>
+    <StringProperty Name="AllowUnsafeBlocks"  Default="False"  DisplayName="Allow unsafe code"/>
+    <StringProperty Name="Optimize" DisplayName="Optimize"/>
+    <EnumProperty Name="WarningLevel" DisplayName="Warning Level"/>
+    <StringProperty Name="NoWarn" DisplayName="Supress Warning"/>
+    <BoolProperty Name="TreatWarningsAsErrors" Visible="True"  Default="False" Description="Treat warnings as errors" />
+    <StringProperty Name="OutputPath" DisplayName="Output Path"/>
+    <StringProperty Name="DocumentationFile" DisplayName="Documentation file"/>
+    <EnumProperty Name="GenerateSerializationAssemblies" DisplayName="Generate serialization assemblies">
+        <EnumValue Name="Auto" DisplayName="Auto" IsDefault="True" />
+        <EnumValue Name="On" DisplayName="On" />
+        <EnumValue Name="Off" DisplayName="Off" />
+    </EnumProperty>
+    <EnumProperty Name="LanguageVersion" DisplayName="Language version"/>
+    <EnumProperty Name="ErrorReport" DisplayName="Error report"/>
+    <EnumProperty Name="DebugInfo" DisplayName="Debug Info">
+        <EnumProperty.DataSource>
+            <DataSource Persistence="ProjectFile" PersistedName="DebugType" />
+        </EnumProperty.DataSource>
+    </EnumProperty>
+    <StringProperty Name="DebugSymbols" DisplayName="Debug symbols"/>
+    <EnumProperty Name="FileAlignment" DisplayName="File Alignment"/>
```

Comment:
![image](https://cloud.githubusercontent.com/assets/12449387/17013658/cc7fe644-4ed4-11e6-9b72-4e33f53614a4.png)

FileAlignment and Language are list of enum values.